node/src/components/consensus/era_supervisor.rs

```diff
@@ -149,11 +151,11 @@ where
             .expect("failure to setup and register ConsensusMetrics");
         let genesis_start_time = protocol_config.timestamp;
 
-        let mut era_supervisor = Self {
+        let era_supervisor = Self {
```

Comment:
I'd assumed `next_block_height` would need to be set correctly here too, but that change can wait until my PR lands if you prefer?

Reply:
I think you're right, but I think waiting for your PR is a good idea - in the current state of the code there is no reasonable source of the correct next block height, as far as I know.